### DISCUSSION TITLE
关于 Post 依赖子文件夹形式

### DISCUSSION THREAD

**Discussion by xuejiaW** [CaiJimmy/hugo-theme-stack#676] [Q&A] [answered]
下载了 exampleSite 及自己尝试后发现，对于新建的文章，必须以`content/Post/<subFolder>/index.md` 的地址创建，如 `\content\post\math-typesetting\index.md` 

有没有哪可以配置成单独 md 的形式而不依赖于文件夹，如 `\content\post\math-typesetting.md`

**Comment by CaiJimmy** [comment, marked as answer]
> 有没有哪可以配置成单独 md 的形式而不依赖于文件夹，如 \content\post\math-typesetting.md

参考：https://gohugo.io/content-management/page-bundles/

`\content\post\math-typesetting.md` 也可以是一篇文章（内容和 `index.md` 一样），不需要单独配置。但它不是一个 Page Bundle，也就无法插入本地图片。
